packages/svelte-ux/src/routes/+layout.svelte

```diff
@@ -213,14 +214,26 @@
 <!-- Set theme before anything renders (even when SSR is in use) -->
 <ThemeInit />
 
-<AppLayout>
+<AppLayout headerHeight={96}>
   <svelte:fragment slot="nav">
     <NavMenu />
     <!-- Spacer -->
     <div class="h-4"></div>
   </svelte:fragment>
 
-  <AppBar {title}>
+  <AppBar {title} class="pt-8">
+    <div
+      class="fixed top-0 left-0 w-full h-8 bg-primary-700 border-b border-primary-800/50 shadow flex gap-2 items-center justify-center text-sm font-medium"
+    >
+    <!-- Maybe remove the Svelte 5 if its not done yet -->
```

Comment:
Yeah, let's remove `Svelte 5 and` since it' not been migrated yet (but fully supports with Svelte 3-5).

Also, let's tweak this to read:
"Svelte UX for Tailwind 4 released!"

I typically write "Svelte UX" instead of "Svelte-UX unless it's the package name, and it should work with for all TW `4.x` releases